cpp/source/rest/private/AccountBalance.hpp

```diff
@@ -12,6 +12,13 @@ namespace Bitwyre::Rest::Private {
       return "/private/account/spotbalance";
     }
 
+    using Callback = std::function<void(const AccountBalanceResponse&)>;
```

Comment:
Please make sure to put the type declarations at the beginning

Reply:
done for all APIs